src/components/Books.js

```diff
@@ -0,0 +1,3 @@
+const Books = () => <h2>Books List Under construction</h2>;
```

Comment:
- [ ] Books should have a form for adding a book (no functionality yet). Let's create a basic form and display it.